docs/resources/sdrs_replication_pair.md

```diff
@@ -0,0 +1,126 @@
+---
+subcategory: "Storage Disaster Recovery Service (SDRS)"
+---
+
+# huaweicloud_sdrs_replication_pair
+
+Manages an SDRS replication pair resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_availability_zones" "test" {}
+data "huaweicloud_sdrs_domain" "test" {}
+data "huaweicloud_vpc" "test" {
+  name = "vpc-default"
+}
+
+resource "huaweicloud_sdrs_protectiongroup" "test" {
+  name                     = "test-name"
+  source_availability_zone = data.huaweicloud_availability_zones.test.names[0]
+  target_availability_zone = data.huaweicloud_availability_zones.test.names[1]
+  domain_id                = data.huaweicloud_sdrs_domain.test.id
+  source_vpc_id            = data.huaweicloud_vpc.test.id
+  description              = "test description"
+}
```

Comment:
```suggestion
variable "source_availability_zone" {}
variable "target_availability_zone" {}
variable "domain_id" {}
variable "vpc_id" {}

resource "huaweicloud_sdrs_protectiongroup" "test" {
  name                     = "test-name"
  source_availability_zone = var.source_availability_zone
  target_availability_zone = var.target_availability_zone
  domain_id                = var.domain_id
  source_vpc_id            = var.vpc_id
  description              = "test description"
}
```

It is better to define parameters as variables to highlight the way resource scripts are written.